scripts/src/lib/Versions.svelte

```diff
@@ -44,32 +111,43 @@
         </select>
     </p>
 
-    <div style="margin-bottom: 15px;">
-        <pre><code>
-minecraft_version={minecraftVersion}
-yarn_mappings={yarnVersion}
-loader_version={loaderVersion}
-loom_version=1.10-SNAPSHOT
-
-# Fabric API
-fabric_version={apiVersion}
-        </code></pre>
+    <CodeTabs {tabs} bind:selectedTab />
 
-				<p><strong>Important Note:</strong> In some cases, such as snapshots or special releases, the <code>fabric-api</code> version might not align perfectly with your Minecraft version.</p>
-				<p>If you encounter issues, double-check the latest release of Fabric API on <a href="https://modrinth.com/mod/fabric-api">Modrinth</a> or <a href="https://minecraft.curseforge.com/projects/fabric/files">CurseForge</a>.</p> 
-      </div>
-			
-			<hr />
+    <p>
+        <strong>Important Note:</strong> In some cases, such as snapshots or special
+        releases, the Fabric API version might not align perfectly with your Minecraft
+        version.
+    </p>
+    <p>
+        If you encounter issues, double-check the latest release of Fabric API
+        on <a href="https://modrinth.com/mod/fabric-api/versions">Modrinth</a>
```

Comment:
if you wanted to, you could even make this open the download modal to the selected Minecraft version such as: https://modrinth.com/mod/fabric-api/versions?version=1.21.5&g=1.21.5#download